src/routes/report-route.tsx

```diff
@@ -40,10 +41,17 @@ function ReportRoute({
   return (
     <Page path={path} title="Daily Auths Report">
       <AgenciesContextProvider>
-        <ReportFilterControlsContext.Provider value={{ start, finish, ial, agency, env }}>
-          <ReportFilterControls path={path} />
+        <ReportFilterContextProvider
+          start={start}
+          finish={finish}
+          ial={ial}
+          agency={agency}
+          env={env}
+          setParameters={defaultSetParameters}
```

Comment:
Would be nice if we could just omit the prop to let it take the default.

Reply:
Yeah I'll have to make the component take separate attributes than the context

Reply:
updated in 9a5e4aee702354f3b2d12f698c045d0525e0f3f1